app/src/main/AndroidManifest.xml

```diff
@@ -16,35 +16,35 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.FixIt"
         android:usesCleartextTraffic="true">
-        <activity android:name=".UI.StaticPage.TermsandConditionsActivity"></activity>
-        <activity android:name=".UI.StaticPage.PrivacyPolicyActivity" />
-        <activity android:name=".UI.StaticPage.AboutUSActivity" />
-        <activity android:name=".UI.Chat.ChatActivity" />
-        <activity android:name=".UI.MainActivity.SplashScreen">
+        <activity android:name=".StaticPage.TermsandConditionsActivity"></activity>
+        <activity android:name=".StaticPage.PrivacyPolicyActivity" />
+        <activity android:name=".StaticPage.AboutUSActivity" />
+        <activity android:name=".Chat.ChatActivity" />
+        <activity android:name=".MainActivity.SplashScreen"/>
+        <activity android:name=".Packages.SubscribePackageActivity" />
+        <activity android:name=".Invoice.InvoiceActivity" />
+        <activity android:name=".ShowProfile.ShowProfileActivity" />
+        <activity android:name=".MyProfile.MyProfileActivity" />
+        <activity android:name=".Contract.ContractScreenShotActivity" />
+        <activity android:name=".PaymentTypes.Paypal.PaypalActivity" />
+        <activity android:name=".InProgress.InProgressActivity" />
+        <activity android:name=".Contract.DisplayAllMillestonesActivity" />
+        <activity android:name=".Contract.DisplayContractActivity" />
+        <activity android:name=".Milestones.MilestonesActivity" />
+        <activity android:name=".Contract.CreateContractActivity" />
+        <activity android:name=".Bids.BidsListActivity" />
+        <activity android:name=".Bids.CreateBidActivity" />
+        <activity android:name=".Projects.ProjectActivity" />
+        <activity android:name=".AddWork.AddWorkActivity" />
+        <activity android:name=".Auth.AuthActivity" />
+        <activity android:name=".Home.HomeActivity" />
+        <activity android:name=".MainActivity.MainActivity" >
             <intent-filter>
```

Comment:
I think it's fine to delete `MainActivity` and keep `SplashScreen` as launcher activity. Since `mainActivity` doesn't do anything but navigating directly to `splashScreen`